manifest/armv7l/a/avahi.filelist

```diff
@@ -1,21 +1,9 @@
-/usr/local/bin/avahi-browse
-/usr/local/bin/avahi-browse-domains
-/usr/local/bin/avahi-publish
-/usr/local/bin/avahi-publish-address
-/usr/local/bin/avahi-publish-service
-/usr/local/bin/avahi-resolve
-/usr/local/bin/avahi-resolve-address
-/usr/local/bin/avahi-resolve-host-name
-/usr/local/bin/avahi-set-host-name
```

Comment:
What happened to all the binaries?

Reply:
All the gtk requiring binaries have been removed in service of making avahi work w/o a gui.